<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ODMR</class>
 <widget class="QWidget" name="ODMR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ODMR</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="measTab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportButton">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="saveconfocalBox">
           <property name="text">
            <string>Save confocal position</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="6" column="1">
          <widget class="QSpinBox" name="bnumBox">
           <property name="suffix">
            <string/>
           </property>
           <property name="prefix">
            <string>burst num: </string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="laserdelayBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="prefix">
            <string>laser delay: </string>
           </property>
           <property name="suffix">
            <string> ns</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QSpinBox" name="numBox">
           <property name="suffix">
            <string/>
           </property>
           <property name="prefix">
            <string>num: </string>
           </property>
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>101</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="backgroundBox">
           <property name="text">
            <string>Measure Background Data</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QDoubleSpinBox" name="mwdelayBox">
           <property name="prefix">
            <string>mw delay: </string>
           </property>
           <property name="suffix">
            <string> ns</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="cwButton">
           <property name="text">
            <string>CW</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QDoubleSpinBox" name="gatedelayBox">
           <property name="prefix">
            <string>gate delay: </string>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Micro Wave Modulation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="amdepBox">
           <property name="prefix">
            <string>am depth: </string>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="fmdevBox">
           <property name="prefix">
            <string>fm deviation: </string>
           </property>
           <property name="suffix">
            <string> kHz</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Micro Wave</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="powerBox">
           <property name="prefix">
            <string>power: </string>
           </property>
           <property name="suffix">
            <string> dBm</string>
           </property>
           <property name="minimum">
            <double>-65.000000000000000</double>
           </property>
           <property name="maximum">
            <double>54.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QDoubleSpinBox" name="triggerwidthBox">
           <property name="prefix">
            <string>trigger width: </string>
           </property>
           <property name="suffix">
            <string> ns</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDoubleSpinBox" name="bgdelayBox">
           <property name="prefix">
            <string>bg delay: </string>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="amlogBox">
           <property name="text">
            <string>am log</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="startBox">
           <property name="prefix">
            <string>start: </string>
           </property>
           <property name="suffix">
            <string> MHz</string>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>10.000000000000000</double>
           </property>
           <property name="maximum">
            <double>3000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>2740.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="sweepsBox">
           <property name="suffix">
            <string> sweeps (0 for inf)</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="mwcontBox">
           <property name="text">
            <string>Continue MW irradiation after stop</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="delayBox">
           <property name="prefix">
            <string>delay: </string>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="stepLabel">
           <property name="text">
            <string>step: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Detection</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="stopBox">
           <property name="prefix">
            <string>stop: </string>
           </property>
           <property name="suffix">
            <string> MHz</string>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>10.000000000000000</double>
           </property>
           <property name="maximum">
            <double>3000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>3000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="windowBox">
           <property name="prefix">
            <string>time window: </string>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QDoubleSpinBox" name="laserwidthBox">
           <property name="prefix">
            <string>laser width: </string>
           </property>
           <property name="suffix">
            <string> ns</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="pulseButton">
           <property name="text">
            <string>Pulse</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QDoubleSpinBox" name="mwwidthBox">
           <property name="prefix">
            <string>mw width: </string>
           </property>
           <property name="suffix">
            <string> ns</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="sgmodBox">
           <item>
            <property name="text">
             <string>no</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>iq</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>fm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>am</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="pulseLabel">
         <property name="text">
          <string>total window: 0.00 single window: 0.00 (rate: 0.00)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Additional PD Parameters</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="ParamTable" name="paramTable"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fiTab">
      <attribute name="title">
       <string>Fit</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="peaksTab">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <attribute name="title">
       <string>Peaks</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParamTable</class>
   <extends>QTableWidget</extends>
   <header>mahos.gui.param</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>saveconfocalBox</tabstop>
  <tabstop>powerBox</tabstop>
  <tabstop>startBox</tabstop>
  <tabstop>stopBox</tabstop>
  <tabstop>numBox</tabstop>
  <tabstop>sweepsBox</tabstop>
  <tabstop>mwcontBox</tabstop>
  <tabstop>sgmodBox</tabstop>
  <tabstop>fmdevBox</tabstop>
  <tabstop>amdepBox</tabstop>
  <tabstop>amlogBox</tabstop>
  <tabstop>backgroundBox</tabstop>
  <tabstop>delayBox</tabstop>
  <tabstop>bgdelayBox</tabstop>
  <tabstop>cwButton</tabstop>
  <tabstop>pulseButton</tabstop>
  <tabstop>windowBox</tabstop>
  <tabstop>gatedelayBox</tabstop>
  <tabstop>laserdelayBox</tabstop>
  <tabstop>laserwidthBox</tabstop>
  <tabstop>mwdelayBox</tabstop>
  <tabstop>mwwidthBox</tabstop>
  <tabstop>bnumBox</tabstop>
  <tabstop>triggerwidthBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
